File: FinalProject/NoteFrontend/src/scripts/json/returncode.ts
/*
 * This class is an easy way to parse and work with the return codes from the API
 */
export class ReturnCode {
    Code: number;
    Name: string;
    Description: string;
  
    constructor(code, name, description) {
      this.Code = code;
      this.Name = name;
      this.Description = description;
    }
  }

File: FinalProject/NoteAPI/Classes/Utilities/ReturnCode.cs
namespace NoteAPI.Classes
{
    public class ReturnCode
    {
        public int Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }

        public ReturnCode(int code, string name, string description)
        {
            Code = code;
            Name = name;
            Description = description;
        }
    }
}
